src/client/utils/validatejs/i18n/de.js

```diff
@@ -63,6 +70,11 @@ validatejs.invalid.globalLocationNumber.message = "Der Wert ist keine gültige G
 validatejs.invalid.iban.message = "Der Wert ist kein gültiger IBAN";
 validatejs.invalid.bic.message = "Der Wert ist kein gültiger BIC";
 validatejs.invalid.swiftCode.message = "Der Wert ist kein gültiger SWIFT-Code";
+validatejs.duplicate.registerationNumber.message = "Ein Unternehmen existiert bereits mit dieser Handelsregisternummer, Stadt und Land.";
+validatejs.duplicate.taxIdNumber.message = "Ein Unternehmen existiert bereits mit dieser Steueridentifikationsnummer und Land.";
```

Comment:
This would be better understandable: **Ein Unternehmen existiert bereits mit dieser Steueridentifikationsnummer und dem Land.**